Content.Shared/_CM14/Scoping/SharedScopeSystem.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Linq;
+using Content.Shared.Actions;
+using Content.Shared.Hands;
+using Content.Shared.Hands.Components;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Popups;
+using Content.Shared.Toggleable;
+using Robust.Shared.Network;
+using Robust.Shared.Timing;
+
+namespace Content.Shared._CM14.Scoping;
+
+public abstract partial class SharedScopeSystem : EntitySystem
+{
+    [Dependency] private readonly SharedActionsSystem _actionsSystem = default!;
+    [Dependency] private readonly ActionContainerSystem _actionContainer = default!;
+    [Dependency] private readonly SharedTransformSystem _transformSystem = default!;
+    [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
+    [Dependency] private readonly SharedPopupSystem _popupSystem = default!;
+    [Dependency] private readonly IGameTiming _gameTiming = default!;
+    [Dependency] private readonly INetManager _net = default!;
+
+    /// <inheritdoc/>
+    public override void Initialize()
```

Comment:
You don't need the <inheritdoc/> here, I think all IDEs should infer it. The doc doesn't have particularly useful information anyways.
```suggestion
    public override void Initialize()
```

Reply:
Fixed